test/patterns/supermethod/test_supermethod.py

```diff
@@ -37,7 +37,7 @@ def test_empty(self):
     def test_instance_of(self):
         # It has 2 matches in anonymous class!
         file = str(Path(self.cur_file_dir, 'Anonymous.java'))
-        self.assertEqual(len(self.testClass.value(file)), 2)
+        self.assertEqual(len(self.testClass.value(file)), 1)
```

Comment:
Please explain why do we change tests?

Reply:
Test are changed, because, for the previous version of this pattern, as you noticed, code brings repetition of code lines. That is why I used set in order to resolve it